pkgs/servers/teleport/generic.nix

```
@@ -116,6 +117,14 @@ let
           ]
       );
 
+    patches = lib.optional (lib.versionAtLeast version "16") [
+      (fetchpatch {
+        name = "disable-wasm-opt-for-ironrdp.patch";
+        url = "https://github.com/gravitational/teleport/commit/994890fb05360b166afd981312345a4cf01bc422.patch?full_index=1";
+        sha256 = "sha256-Y5SVIUQsfi5qI28x5ccoRkBjpdpeYn0mQk8sLO644xo=";
```

```suggestion
        hash = "sha256-Y5SVIUQsfi5qI28x5ccoRkBjpdpeYn0mQk8sLO644xo=";
```
See https://github.com/NixOS/nixpkgs/issues/325892

---

Done, thanks for the suggestion.